### PR TITLE
Style change

### PR DESCRIPTION
**Pull request by Sh11ntar0**
Changed the Style Sheet.
Please check it.

### CODE REVIEW

**Review by BlackStone8960 — APPROVED** [review]
Looks good, thank you for doing such a good job!

File: src/components/AboutPage/AboutPage.js [hunk 1]
```diff
@@ -1,148 +1,324 @@
...
+              viewBox="0 0 24 24"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M20 7V17C20 18.1046 19.1046 19 18 19H8"
+                stroke="#a0a0a0"
+                stroke-width="1.5"
+                stroke-linecap="round"
+                stroke-linejoin="round"
```
Comment (miyabitanimchi) [inline src/components/AboutPage/AboutPage.js:239]:
```suggestion
                strokeWidth="1.5"
                strokeLinecap="round"
                strokeLinejoin="round"
```

File: src/components/AboutPage/AboutPage.js [hunk 2]
```diff
@@ -1,148 +1,324 @@
...
+              />
+              <rect
+                x="5.75"
+                y="4.75"
+                width="10.5"
+                height="10.5"
+                rx="1.25"
+                stroke="#a0a0a0"
+                stroke-width="1.5"
```
Comment (miyabitanimchi) [inline src/components/AboutPage/AboutPage.js:248]:
```suggestion
                strokeWidth="1.5"
```

**Review by miyabitanimchi — CHANGES_REQUESTED** [review]
Thank you for your great job! There was one thing I noticed, but the errors below occur when you go to the about page.
スクリーンショット 2021-12-11 午後7 04 43
スクリーンショット 2021-12-11 午後7 04 49
スクリーンショット 2021-12-11 午後7 04 56

As the errors say, in React they should be `strokeWidth`, `strokeLinecap` and `strokeLineJoin` respectively.
Please check the suggested changes below as well, thank you!

**Comment by Sh11ntar0** [comment]
I fixed the errors that Miyabi told me about.

**Pull request merged**